hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1641,7 +1641,7 @@ void optimizeARM(Vunit& unit, const Abi& abi, bool regalloc) {
   simplify(unit);
 
   lowerForARM(unit);
-
+  reuseImmq(unit);
```

Comment:
@swalk-cavium—Do you have any stats comparing placement of this pass before/after `foldImms()`?